src/playlist.rs

```diff
@@ -584,6 +592,8 @@ pub struct MediaSegment {
     pub program_date_time: Option<String>,
     /// `#EXT-X-DATERANGE:<attribute-list>`
     pub daterange: Option<String>,
+    /// `#EXT-` that are unknown or customized
+    pub unknown: Option<Vec<ExtTag>>,
```

Comment:
It's fine to just use a `Vec<ExtTag>` instead of `Option<Vec<ExtTag>>` here

Reply:
Actually, after looking at #19, I think it's better to move the unkown tags up to `MasterPlaylist` and `MediaPlaylist`. Let me know if you want some help or if I can push the changes to your branch

Reply:
thank you for the feedback @rutgersc ! If you want to make the change, please feel free. I can have a look later if this is still open.

Reply:
Alright, i've pushed the changes 👍